src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <a href="index.html">
+          <img
+            class="logo_png"
+            src="images/logo.png"
+            alt="moyo-logo"
+          />
+        </a>
+
+        <nav>
+          <ul class="nav__list">
+            <li class="nav nav__item">
+              <a
+                class="nav__link is-active blue"
+                href="url"
+              >
+                apple
+              </a>
+            </li>
+            <li class="nav nav__item">
+              <a
+                class="nav__link"
+                href="url"
+              >
+                samsung
+              </a>
+            </li>
+            <li class="nav nav__item">
+              <a
+                class="nav__link"
+                href="url"
```

Comment:
The `href` attribute contains a placeholder 'url'. Replace it with the actual URL you want this link to point to.